app/mails/management/commands/sendmail.py

```diff
@@ -109,3 +58,63 @@ def handle(self, *args, **kwargs):
                 mail.delete()
 
         smtp.quit()
+
+
+def connect_to_smtp():
+    try:
+        smtp = smtplib.SMTP(settings.EMAIL_HOST)
+        smtp.starttls()
+        smtp.login(settings.EMAIL_HOST_USER, settings.EMAIL_HOST_PASSWORD)
+    except:
+        return
+
+
+def handle_mulitpart_message(message, mail, text, msg):
+    charset = message.msg.get_content_charset()
+
+    if message.msg.is_multipart():
+        add_text = MIMEText(text, "plain", "utf-8")
+        if message.msg.get_content_maintype == "multipart/signed":
+            # If it's a signed message, only take first payload
+            msg = MIMEMultipart()
+            orig = message.msg.get_payload(0)
+            msg.attach(orig)
+            msg.attach(add_text)
+        else:
+            msg = MIMEMultipart()
+            msg.attach(message.msg)
+            msg.attach(add_text)
+
+    else:
+        msg = MIMEText(
+            "\n\n".join((message.msg.get_payload(), str(text))), "plain", charset,
+        )
+
+
+def handle_attachment(message, mail, text, msg):
```

Comment:
Try to find some better naming for the functions than `handle_*`. I like how you split out the functionality, but the function names should reflect closely what's happening "inside".

Reply:
Is it better now?